Library/Homebrew/cmd/update.sh

```diff
@@ -314,7 +314,7 @@ homebrew-update() {
       *)
         odie <<EOS
 This command updates brew itself, and does not take formula names.
-Use 'brew upgrade $@' instead.
+Use \`brew upgrade $@\` instead.
```

Comment:
Are you sure these need escaped?

Reply:
Yes; it prints `Use  instead.` otherwise.

Reply:
Ah, missed this was Bash.